scripts/lint_site.sh

```diff
@@ -18,10 +18,22 @@ set -e
 
 FAILED=0
 
+BRANCH="${BRANCH:-master}"
+
```

Comment:
This doesn't look like it will do the right thing when running `make lint` locally.

```suggestion
```

Reply:
Correct, locally means that we would default to master which should be where the changes are being made against. But for simplicity I will remove this in favor of below.